authors/apps/articles/renderers.py

```diff
@@ -0,0 +1,55 @@
+import json
```

Comment:
Have you rebased develop? This file was removed on a previous PR

Reply:
I have but let me check the file